runtime/calamari/src/weights/frame_system.rs

```diff
@@ -56,7 +55,7 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> frame_system::WeightInfo for SubstrateWeight<T> {
 	fn remark(b: u32, ) -> Weight {
-		(0 as Weight)
+		(353_000 as Weight)
```

Comment:
This is odd.
I ran this multiple times and the results are not consistent. Between 0 and 500_000
```
./target/release/manta \
 benchmark \
 --chain=calamari-dev \
 --execution=wasm \
 --wasm-execution=compiled \
 --pallet=frame_system \
 --extrinsic=remark \
 --steps=50 \
 --repeat=20 \
 --template=.github/resources/frame-weight-template.hbs \
 --output=check
 ```

It could be a bug in the benchmarking system or expected, not sure yet.

Reply:
It's a problem, but doesn't affect calamari. A normal tx like `balances.transfer` will cost 0.627KMA, remark costs 0.626KMA. But perhaps we need to write unit tests for benchmark results. 
Like every time we benchmark runtime, record the benchmark results. 
When next benchmark, compare current result with last one.

Reply:
i would recommend we push these to a timeseries DB and a grafana line graph for manual trend visualization and automated alerts on large changes.
Not sure how far the plans for grafana have advanced (i.e. when we'll have an instance to use), I think @rishabhfsd was working on it

Reply:
This is Parity's visualization https://user-images.githubusercontent.com/10380170/151775062-e22f6d5a-5c7b-4649-83d5-fb836d0f1324.png